src/main/java/net/ossindex/gradle/OssIndexPlugin.java

```diff
@@ -61,11 +83,24 @@ private Proxy getProxy(Project project, String scheme) {
     }
 
     private void doAudit(Task task) {
+
+        if (this.settings == null) {

```

Comment:
As mentioned above, I think that this block can go away and you can just instantiate the settings where you use them on line 92...